docs/_templates/layout.html

```diff
@@ -1,6 +1,11 @@
 {%- extends "!layout.html" %}
 
 {%- block document %}
+  <div class="announcement-banner-wrapper">
+    <div class="announcement-banner">
+      <p>{{ announcement_message | safe }}</p>
+    </div>
+  </div>
```

Comment:
I wonder if we could use something like 

```
.. postlist:: 1
```

perhaps with a check on whether Portland is active using `flagpostconf: False` to just show the last news post?

Reply:
It's kind of a pain to bring in content like that, so I think driving it from the config is probably easiest for now?